internal/plans/changes.go

```diff
@@ -301,9 +301,10 @@ func (rc *ResourceInstanceChange) Simplify(destroying bool) *ResourceInstanceCha
 				Private:      rc.Private,
 				ProviderAddr: rc.ProviderAddr,
 				Change: Change{
-					Action: Delete,
-					Before: rc.Before,
-					After:  cty.NullVal(rc.Before.Type()),
+					Action:    Delete,
+					Before:    rc.Before,
+					After:     cty.NullVal(rc.Before.Type()),
+					Importing: rc.Importing,
```

Comment:
I think `Changes.Empty()` should take this into account too for consistency, even though it won't be directly used right now.